build/webpack.js

```diff
@@ -180,7 +180,11 @@ export default async function getBaseWebpackConfig (dir: string, {dev = false, i
       hotUpdateMainFilename: 'static/webpack/[hash].hot-update.json',
       // This saves chunks with the name given via `import()`
       chunkFilename: isServer ? `${dev ? '[name]' : '[contenthash]'}.js` : `static/chunks/${dev ? '[name]' : '[contenthash]'}.js`,
-      strictModuleExceptionHandling: true
+      strictModuleExceptionHandling: true,
+      // Point sourcemap entries to original disk location
+      devtoolModuleFilenameTemplate: info => dev
+        ? path.resolve(info.absoluteResourcePath).replace(/\\/g, '/')
```

Comment:
This will break source-maps in chrome 🤔

Reply:
@timneutkens we could also could fix the invalid paths coming from `react-error-overlay` inside the `errorOverlayMiddleware`. Otherwise me might need to wait for create-react-app@2.

```js
import url from 'url'
import launchEditor from 'launch-editor'
import path from 'path'

export default function errorOverlayMiddleware (req, res, next) {
  if (req.url.startsWith('/_next/development/open-stack-frame-in-editor')) {
    const query = url.parse(req.url, true).query
    const lineNumber = parseInt(query.lineNumber, 10) || 1
    const colNumber = parseInt(query.colNumber, 10) || 1

    // Resolve invalid paths coming from react-error-overlay
    const webpackSchemeRegexp = /^webpack:\/\//
    const fileName = webpackSchemeRegexp.test(query.fileName)
      ? path.resolve(query.fileName.replace(webpackSchemeRegexp, ''))
      : query.fileName

    launchEditor(`${fileName}:${lineNumber}:${colNumber}`)
    res.end()
  } else {
    next()
  }
}
```

Reply:
The overlay needs a lot of love, Dan and I should have a good chunk of time to spend on CRA starting around mid-September.
We're going to be re-reviewing a lot of this code, so I'll be sure to keep Next.js in mind and how you guys are using it.

Thanks for the ping!